scripts/download_data.py

```diff
@@ -500,11 +512,13 @@ def parseArgs():
 @utils.Timed()
 def main(args):
     downloaderClass = sources[args.source]
-    downloader = downloaderClass(args)
-    downloader.downloadVcfs()
-    downloader.downloadBams()
-    downloader.downloadFastas()
-    downloader.cleanup()
+    try:
+        downloader = downloaderClass(args)
+        downloader.downloadVcfs()
+        downloader.downloadBams()
+        downloader.downloadFastas()
+    finally:
+        downloader.cleanup()
```

Comment:
What's the problem with leaving the checkpoint functionality in?  The network connection could always drop in the middle of downloading.  And, although I hope for network improvements in this version of htslib, I doubt it has been purged of all bugs.